hazelcast/src/main/java/com/hazelcast/map/IMap.java

```diff
@@ -73,7 +73,7 @@
  * {@code entrySet}, return an <b>immutable</b> collection clone of the values.
  * The collection is <b>NOT</b> backed by the map, so changes to the map are
  * <b>NOT</b> reflected in the collection.</li>
- * <li>Since Hazelcast is compiled with Java 1.6, we can't override default
+ * <li>Since Hazelcast is compiled with Java 11, we can't override default
```

Comment:
I'll redo this section. We compile with JDK 8 now and the second part of the sentence does not hold anymore. Basically, the whole section can now be removed.